x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/compute_usage.ts

```diff
@@ -139,12 +131,12 @@ export async function getComputeUsage({
                 { defaultMessage: 'Compute usage' }
               ),
               key: 'compute_usage',
-              type: 'linemark',
+              type: 'bar',
               overallValue: calculateComputeUsageGBSeconds({
                 faasBilledDuration: aggregations?.avgFaasBilledDuration.value,
                 totalMemory: aggregations?.avgTotalMemory.value,
               }),
-              color: getVizColorForIndex(0, theme),
+              color: getVizColorForIndex(1, theme),
```

Comment:
If we want a specific color, it's better to specify it directly
```suggestion
              color: theme.euiColorVis1
```

`getVizColorForIndex` is for generating a range of colors dynamically